packages/core/tests/proofs-auto-accept.test.ts

```diff
@@ -0,0 +1,206 @@
+import type { Agent, ConnectionRecord, PresentationPreview } from '../src'
+import type { CredDefId } from 'indy-sdk'
+
+import {
+  AutoAcceptProof,
+  ProofState,
+  ProofAttributeInfo,
+  AttributeFilter,
+  ProofPredicateInfo,
+  PredicateType,
+} from '../src'
+
+import { setupProofsTest, waitForProofRecord } from './helpers'
+import testLogger from './logger'
+
+describe('Auto accept present proof', () => {
```

Comment:
Shouldn't we have tests also for accept never?

Reply:
It seems we're testing just a happy path.

Reply:
Since I assumed the default is `Never`, the `proofs.test.ts will error if it does any automatic response. I could add the `never` clause as a test, but it would feel a bit redundant.